test/regress/regress0/eqrange1.smt2

```diff
@@ -0,0 +1,15 @@
+(set-logic QF_AUFBV)
+(set-option :arrays-exp true)
+(set-option :quiet true) ; Suppress Warning
```

Comment:
what warning?

Reply:
This is to suppress a warning about check-model involving quantifiers I believe.

Reply:
I see. In this case it might be good to add a comment to the regression.